### DISCUSSION TITLE
Installing on Linux

### DISCUSSION THREAD

**Discussion by mkrueger** [mkrueger/icy_tools#3] [Q&A]
It's probably the easiest OS to build the icy_tools on however atm I don't have any installer. 

How should the icy tools to be installed on Linux?  I've basically no idea what's the best approach should be.

**Comment by mkrueger** [comment]
I added build chains for deb files built on latest ubuntu again - let me know if these work for you.

**Comment by AppleSeedTrees** [comment]
Hello mkruger, I was able to compile the tools in Arch and get them running without any issue, but I ran into an issue with SSH in ICY_TERM I get "No Common Algorithm". I will have to look at the code to see where it is pulling the SSH keys and what I need to fix the keys for SSH. I really don't know what I am doing in that area, but I will do what I can. If I get it going I can add it to the AUR or you can. If need help. Let me know. All the other programs runs great and Icy_Term works perfectly except for SSH.

**Comment by mkrueger** [comment]
I updated the ssh lib - should work now.
